stories/calendar/props-table.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { ApiTable } from "../storybook-common/api-table";
+import { ApiTableSectionProps } from "../storybook-common/api-table/types";
+
+const DATA: ApiTableSectionProps[] = [
+    {
+        name: "Calendar specific props",
+        attributes: [
+            {
+                name: "value",
+                description: (
+                    <>
+                        The value of the start date in string based
+                        format.&nbsp;
+                        <code>YYYY-MM-DD</code>
+                    </>
+                ),
+                propTypes: ["string"],
```

Comment:
you're missing the default which is today?

Reply:
??

the `value` props for the value from the server. if there is not exist, not selected the calendar after component mounted.